components/GothamistHomepageTopper.vue

```diff
@@ -54,7 +57,7 @@ const latestArticles = computed(() => {
     </div>
     <div class="col-12 xl:col-4 flex flex-column justify-content-end">
       <hr class="black mb-3 xl:hidden" />
-      <SponsorPlacement />
+      <!-- <SponsorPlacement /> -->
```

Comment:
Why not just delete this?

Reply:
Actually, we can probably delete the component too.

Reply:
And the logo images too.

Reply:
ok sounds good